wdl-lint/tests/lints/multiple-eof-newline/source.wdl

```diff
@@ -0,0 +1,8 @@
+# This is a test of multiple EOF newlines lint
+
+version 1.1
+
+workflow test {
+
+}
+
```

Comment:
```suggestion



```
I'd like to make sure that the `span` reported covers _every_ superfluous newline.